### DISCUSSION TITLE
Installation hangs then fails

### DISCUSSION THREAD

**Discussion by Riprock** [Security-Onion-Solutions/securityonion#7795] [Unsupported Versions] [answered]
I am going to install a new instance of security onion in my lab and when I am in the process of going thorough the installer, it reaches 95% where it says verifying installation, hangs then fails to where it tells me to check /root/errors.log but when I check the file, theres nothing in the file. Checking sosetup.log, The last log item is "Removing so-setup permission entry from sudoers file" Any help would be appreciated

**Comment by dougburks** [comment, marked as answer]
What options are you choosing in the installer?

What are the specs of your system?
https://docs.securityonion.net/en/2.3/hardware.html#minimum-specs

**Reply by Riprock** [reply]
![image](https://user-images.githubusercontent.com/8990377/163697492-756e2077-fba0-4b6f-bfed-55991a9bc4ad.png)
Those are system specs.
Im going with all default options and when it asks about what Im doing, Im doing standalone install. eth0 is my management and the other 3 interfaces are all monitor

**Reply by dougburks** [reply]
12GB RAM is usually sufficient for an EVAL installation. However, a STANDALONE installation likely requires more especially when sniffing multiple interfaces.

**Reply by Riprock** [reply]
Ok I will Bump up the ram
